doc/sphinx/addendum/extraction.rst

```diff
@@ -48,6 +48,7 @@ All the following commands produce real ML files. User can choose to
 produce one monolithic file or one file per Coq library.
 
 .. cmd:: Extraction @string {+ @qualid }
+   :name: _
```

Comment:
How about combining this with `Extraction` shown above into a single description?  If you use `_`, it won't be easily found from the index.